TASVideos/Pages/Forum/Topics/_TopicPost.cshtml

```diff
@@ -54,21 +54,18 @@
 	<cardbody class="p-2">
 		<row>
 			<div class="forum-profile-section col-lg-2 col-md-3 col-4">
-				<div class="text-center">
+				<div class="text-center" style="margin-left: -8px;">
```

Comment:
using px should be avoided (we have classes like ml-1, mt-2, etc).  Using negative pixels needs one hell of a justification